.storybook/preview-body.html

```diff
@@ -1,9 +1,9 @@
 <style>
   html {
-    font-size: var(--legacy-size-font-base);
+    font-size: var(--eds-font-size-base);
   }
 
   body {
-    font-family: Arimo, sans-serif;
```

Comment:
Was this not working before?

Reply:
oh I meant to swap in the new token 🤦‍♀️